docs/CONTRIBUTING.md

```diff
@@ -47,9 +47,11 @@ and tools contained in LoopBack repositories.
 
 ## [Documentation](http://loopback.io/doc/en/contrib/doc-contrib.html)
 
-LoopBack documentation is sourced in the
-[strongloop/loopback.io](https://github.com/strongloop/loopback.io) GitHub
-repository.
+LoopBack documentation is sourced in both the
```

Comment:
Just wondering... if all the LoopBack 4 documentation are located in the `loopback-next` repo, maybe we don't even need to mention `loopback.io` at all?

Reply:
I think she meant LoopBack in general not just v4 which would make sense.

Reply:
Well, not all documentation is located in loopback-next. Many parts of documentation for contributors is still at loopback.io only - e.g. bug reporting instructions.

It would be great if we could add a short explanation here, to let readers understand where to look for what.

Also loopback-next is a large monorepo, please add a link to `docs/site` to make it easier for users to find doc files.